docs/tutorials/running_on_lfrqa.md

```diff
@@ -0,0 +1,234 @@
+# Preparing the LFRQA Dataset
+
+## Overview
+
+The **LFRQA dataset** was introduced in the paper [_LFRQA: Large-Scale Few-Shot Retrieval Question Answering_](https://arxiv.org/pdf/2407.13998). It features **1,404 science questions** (along with other categories) that have been human-annotated with answers. This tutorial walks through the process of setting up the dataset for use.
+
+## Download the Annotations
+
+First, we need to obtain the annotated dataset from the official repository:
+
+```bash
+# Clone the RAG-QA Arena repository
+git clone <https://github.com/awslabs/rag-qa-arena>
```

Comment:
Why do you have the `<>` here, why not just use the actual HTTPS git clone URL?

`git clone https://github.com/awslabs/rag-qa-arena.git`